src/Pagination/index.jsx

```diff
@@ -417,6 +417,8 @@ The default is:
 }
 ``` */
   buttonLabels: PropTypes.shape({
+    leftChevron: PropTypes.element,
```

Comment:
```suggestion
    leftChevron: PropTypes.node,
```
(any renderable element)